linode/manager
# M3-5607: Update stories for loading states

## Description

- Remove unused props or props that have no visible UI changes
- Convert to MDX
- Add copy
- Clean up examples
- Removed a couple components that are no longer in use (ie. `InlineTextLoader` and `LoadingText`)

## How to test

- Run `yarn storybook`
- Go to http://localhost:6006/?path=/docs/ui-elements-loading-states-bar-percent--bar-percent
- Check the app for any regressions
